content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -19,7 +19,45 @@ This task shows you how to access external services in three different ways:
 1. Configure [service entries](/docs/reference/config/networking/v1alpha3/service-entry/) to provide controlled access to external services.
 1. Completely bypass the Envoy proxy for a specific range of IPs.
 
-{{< boilerplate before-you-begin-egress >}}
+## Before you begin
+
+*   Setup Istio by following the instructions in the [Installation guide](/docs/setup/).
+
+    {{< warning >}}
```

Comment:
Don't you want to put this warning before the "Before you begin" section, and then use the boilerplate?

Reply:
I really want it at this step of the instructions, so there's no nice way to use the boilerplate (as is) and add the warning IMO.

Reply:
OK, sounds good.